jobbergate-docs/src/developer_guide/ci.rst

```diff
@@ -2,4 +2,76 @@
  Continuous Integration
 ========================
 
-TODO: Write this in concert with Felipe
+Jobbergate uses `GitHub actions <https://github.com/omnivector-solutions/jobbergate/actions>`_ as continuous integration tools. They are described in details in this page.
+
+-------------------
+ Quality Assurance
+-------------------
```

Comment:
For other docs pages we aren't using the top line and extra spacing for sections inside the body.

```suggestion
Quality Assurance
-----------------
```

Reply:
solved in [58f2f91](https://github.com/omnivector-solutions/jobbergate/pull/144/commits/58f2f918da90e960892e55f68c4e15734043d298).